docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -30,22 +30,32 @@ Array.sum array
 ```
 
 #### Parameters
-*array*
-Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
-
+*array* Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
 
 The input array.
 
 
+## Return Value
+
+The resulting sum.
+
 
-**The resulting sum.**
 ## Remarks
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception(v=vs.110).aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
```

Comment:
@ReedCopsey 

Oh, thanks for the version hint! I just noticed, other links seem to use some sort of hash instead of the repo path, is there a reason I should use one or the other?

IIRC I encountered two files with a different convention on the colon. I then checked the current consensus and found that the Chicago Manual of Style is now recommending that punctuation "should appear in the same font (...) as the enclosing text".

I don't remember which file I compared to though; I was checking files from other places since the typography on the ones in the vicinity of this one seemed obviously broken in similar ways (e.g. the output formatting in all-bold concatenated text). I didn't check many others since it didn't seem like something even 1% of readers would care about, so I simply followed the typographic recommendation.

I can leave the colons alone, it's not like I'd have cared if whatever reference file I first used hadn't clashed on this convention.